tests/scripts/pulp_rpm/test_uln_remote.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
The tests will fail unless/until we add ULN_REMOTE_URL, user, and pass, to project-secrets. As a project, pulp doesn't have ULN access - and it wouldn't be appropriate to use a contributor's access to drive our CI :)

I'd like to see the test a) rely on env-vars for user/pass, b) some brief commentary on "you need to set variables ULN_REMOTE_URL, USER, and PASSWORD before this test will work, and c) at the top of the test, if url/user/poass are NOT set, just return true. That'll let us keep the test to run locally when/as we can, but won't hold up CI.

Reply:
I think, we made `exit 3` let the test appear as skipped.

Reply:
Also i do not see the tests _use_ the remote, so i think just CRUD on the remote we can do without a problem.